packages/node-cli/src/commands/ManageAccountCommandGenerator.js

```diff
@@ -0,0 +1,284 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+'use strict';
+
+const BaseCommandGenerator = require('./BaseCommandGenerator');
+const SdkExecutionContext = require('../SdkExecutionContext');
+const { executeWithSpinner } = require('../ui/CliSpinner');
+const SdkOperationResultUtils = require('../utils/SdkOperationResultUtils');
+const CommandUtils = require('../utils/CommandUtils');
+const AccountCredentialsService = require('../services/AccountCredentialsService');
+const NodeTranslationService = require('../services/NodeTranslationService');
+const ManageAccountOutputFormatter = require('./outputFormatters/ManageAccountOutputFormatter');
+const AuthenticationService = require('../core/authentication/AuthenticationService');
+const { ManageAccountActionResult } = require('./actionresult/ManageAccountActionResult');
+const assert = require('assert');
+
+const inquirer = require('inquirer');
+
+const {
+	COMMAND_MANAGE_ACCOUNT: { ERRORS, QUESTIONS, QUESTIONS_CHOICES, MESSAGES },
+	YES,
+	NO,
+} = require('../services/TranslationKeys');
+
+const ACTION = {
+	LIST: 'list',
+	EXIT: 'exit',
+	INFO: 'info',
+	RENAME: 'rename',
+	REMOVE: 'remove',
+	REVOKE: 'revoke',
+};
```

Comment:
It will be nice to group this ACTION list only in one place, because we have the same ones in the ManageAccountOutputFormatter. Maybe in the ManageAccountActionResult, and export it from there?